bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -49,15 +49,16 @@
           'alt': 'Firefox'
       })
     }}
-    {{ resp_img(
+    <div class="ctd-logo-sprite"></div>
+    {# {{ resp_img(
         url='img/firefox/challenge-the-default/logo-ctd.svg',
         optional_attributes={
           'loading': 'eager',
           'height': '24',
           'class': 'ctd-sub-nav-image',
           'alt': 'Challenge the Default'
         }
-    ) }}
+    ) }} #}
```

Comment:
Question/suggestion: should we remove this if it's no longer needed?

Also, seeing as the sprite has no `alt` text now, I wondered if we should add "Challenge the Default" to the alt text of the Firefox logo right next to this? It seems like an easy compromise that at least ensures the campaign name can still be read aloud.

Thoughts?